lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -110,7 +110,7 @@ private function build($data) {
       'id' => isset($northstar_user) ? $northstar_user->id : $data->field_northstar_id_value,
       'first_name' => dosomething_helpers_extract_field_data($drupal_user->field_first_name),
       'last_initial' => $last_initial,
-      'photo' => dosomething_helpers_extract_field_data($drupal_user->photo),
+      'photo' => isset($drupal_user->picture) ? $drupal_user->picture : NULL,
```

Comment:
@chloealee Here's the [`signups/` proxy route](https://github.com/DoSomething/northstar/blob/dev/app/Http/Controllers/SignupController.php#L32-L58) in Northstar.

I think @weerd's solution is a good fix for now, but I do want to think a bit on how we can better handle situations like this so we're not piping requests through an ever-growing number of services when we need info from multiple places.